apps/project/tests/test_apis.py

```diff
@@ -78,6 +77,40 @@ def test_create_project(self):
         self.assertEqual(Project.objects.count(), project_count + 1)
         self.assertEqual(response.data['title'], data['title'])
 
+    def test_check_assessment_template_in_project_create(self):
+        project_count = Project.objects.count()
+        assessment = self.create(AssessmentTemplate)
+        url = '/api/v1/projects/'
+        data = {
+            'title': 'Test project',
+            'data': {'testKey': 'testValue'},
+            'organizations': [
+                {'organization': self.org1.id, 'organization_type': ProjectOrganization.DONOR},
+            ],
+            'has_assessments': True
+        }
+
+        self.authenticate()
+        response = self.client.post(url, data)
+        self.assert_201(response)
+
+        self.assertEqual(Project.objects.count(), project_count + 1)
+        self.assertEqual(response.data['assessment_template'], assessment.id)
+
+        # not providing has_assessments
+        data = {
+            'title': 'Test project',
+            'data': {'testKey': 'testValue'},
+            'organizations': [
+                {'organization': self.org1.id, 'organization_type': ProjectOrganization.DONOR},
+            ],
+            'has_assessments': False
+        }
```

Comment:
This should be enough. Let's also add for None

```suggestion
        data['has_assessments'] = False
```

Reply:
Done